lib/dynamoid_advanced_where/query_materializer.rb

```diff
@@ -23,49 +24,72 @@ def all
     def each(&blk)
       return enum_for(:each) unless blk
 
+      each_page.flat_map{|i, _| i }.each(&blk)
```

Comment:
Since `.each_page` doesn't take a block, it looks like we're trying to just always use `.each`, if so does the `{|i,_}` here make us lose the raw results object needed to pull our continuation tokens out?